README.md

```diff
@@ -1,57 +1,117 @@
+A simple three-tiered application (database, server, client) that is designed to be run locally or depoyable to the BC Government OpenShift cluster with ease.
 
 ## Purpose
 
-Most new project teams working within the BC Developers Exchange environment experience a kick-off period where they need to learn the tools and processes necessary to create and deploy a new application. Even something as seemingly simple as implementing a reference 3-tiered architecture can occupy a great deal of development time if the team doesn't start with a simple boilerplate. OpenShift configuration and deployment itself is a steep learning curve. With all of the new things to learn and absorb, "Sprint Zero" can easily drag out for weeks.
+Most new project teams working within the BC Developers Exchange environment experience a lengthy acclimation period in which they learn about the tools and environment of the lab. While familiar with developing apps, teams are often unaware of how to deploy them to OpenShift. The concept of OpenShift configuration and deployment itself has a steep learning curve. With all of the new things to learn and absorb, *Sprint Zero* can easily drag out for weeks.
 
-This project is intended as a starting point for the team with a Product Owner who wants to start realizing business value soon after the project kicks off. The repo before you does not have everything necessary to solve every problem you will face, but it should work out of the box and allow you to put working features in front of users sooner. 
+This project is intended as a starting point for the team with a Product Owner who wants to start realizing business value as soon as possible after the project kicks off. This repository does not have everything necessary to solve every problem you will face, but it should work out of the box and allow you to add features sooner.
 
 ## Philosophy
-- Everything is Open-Source. 
-- Keep things as simple as possible (you can make it more complicated later if you need to).
-- Automate as much as possible (you can find all the scripts and run them manually if you want to).
-- We should be able to check out, build and run locally with minimal effort (using docker-compose).
+
+- Everything is open-source. 
+- Keep things simple and add complexity when required.
+- Automate repetitive commands while exposing what's being done.
+- Obfuscate the underlying technology as little as possible.
+- Able to clone, build, and run locally with minimal effort.
 - Keep everything compatible with deployment to OpenShift.
-- Reduce technical debt as much as possible and keep cleaning it up as you go.
+- Minimize technical debt.
 
 ## Requirements:
-- Git
-- “Make” (installed by default on OS X and Linux)
+
+- [Git](https://git-scm.com)
+- [make](http://man7.org/linux/man-pages/man1/make.1.html) (installed by default on OS X and Linux)
 - [Docker](https://www.docker.com/)
-- OpenShift CLI
```

Comment:
So I tried going to the link and installing the OpenShift CLI for Mac. It takes me to a login page where I need to provide RedHat credentials. @ashtonmeuser just saved me a bunch of trouble by asking me to install the CLI from brew. Definitely would recommend mentioning the brew option for a noob like myself.